rotkehlchen/chain/ethereum/modules/thegraph/constants.py

```diff
@@ -1,4 +1,5 @@
 from typing import Final
 from rotkehlchen.chain.evm.types import string_to_evm_address
 
+TRANSPARENT_UPGRADABLE_PROXY: Final = string_to_evm_address('0xCa82c7Ce3388b0B5d307574099aC57d7a00d509F')  # noqa: E501
```

Comment:
`TRANSPARENT_UPGRADABLE_PROXY` is a type of contract. The name of the contract being proxied is `L1GraphTokenLockTransferTool`. 

```suggestion
GRAPH_L1_LOCK_TRANSFER_TOOL: Final = string_to_evm_address('0xCa82c7Ce3388b0B5d307574099aC57d7a00d509F')  # noqa: E501
```